### DISCUSSION TITLE
为什么不读取这个目录哇

### DISCUSSION THREAD

**Discussion by fhwlnetwork** [Charles7c/charles7c.github.io#85] [General]
![image](https://github.com/Charles7c/charles7c.github.io/assets/30582975/fbb70758-ff0d-4b87-a179-32c46c969146)
为什么不读取这个目录哇
![image](https://github.com/Charles7c/charles7c.github.io/assets/30582975/8fad568e-7c7e-4046-a708-c592c816d007)

**Comment by Charles7c** [comment]
你注意看下目录结构，我的文章目录结构是在每个分类下以年/月/日来分隔的。目录结构不一致则无法识别，这种情况可以考虑使用一些自定义情况更好的 VitePress 自动生成 SideBar 插件，e.g. vitepress-sidebar，或者根据关键词搜索一下其他的。
